atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -635,14 +637,12 @@ def get_buildroot(self, worker_metadatas):
 
         return buildroots
 
-    def _update_extra(self, extra, metadata, worker_metadatas):
-        extra['image']['autorebuild'] = False
-
+    def _update_extra(self, extra, worker_metadatas):
         if not isinstance(self.workflow.source, GitSource):
             raise RuntimeError('git source required')
 
         try:
-            isolated = str(metadata['labels']['isolated']).lower() == 'true'
+            isolated = self.workflow.user_params['isolated'].lower() == 'true'
```

Comment:
IIUC, it must be string while it was a label, but it is boolean in user_params

Reply:
Fixed